platform/ios/Makefile

```diff
@@ -9,6 +9,12 @@ appstore_api_issuer_id = $(POLYPOD_APPLE_IOS_APPSTORE_API_ISSUER_ID)
 provisioning_profile_dir := ~/Library/MobileDevice/Provisioning\ Profiles
 provisioning_profile_file := cooppolypolypolypod__iOS_App_Store_Distribution.mobileprovision
 
+build_version := $(shell /usr/libexec/PlistBuddy -c "Print :ApplicationProperties:CFBundleShortVersionString" "${archive_path}/Info.plist")
+build_number := $(shell /usr/libexec/PlistBuddy -c "Print :ApplicationProperties:CFBundleVersion" "${archive_path}/Info.plist")
+appstore_app_id := 1562456558
```

Comment:
This is strange - somehow when we upload the build, the App Store figures out which app it belongs to, but here we have to hard code some ID?

Reply:
Using an environment variable should be a bit more flexible (and maybe, not sure, secure)

Reply:
Apple would derive the app id from bundle identifier.
I have specified directly the appstore app id to avoid making an additional request to exchange the bundle identifier for an ID. As, all subsequent API calls need app ID not app bundle identifier.

Thinking over it again, while this would be an additional API request, maybe it would be a better approach to not have this id exposed. Instead the bundle identifier from the Archive could be used, the same way build_version and build_number could be used. I will investigate this option.

Reply:
I think it'd be nicer, but not worth a ton of extra work - it should change rather rarely :D